resotocore/resotocore/cli/command.py

```diff
@@ -4608,6 +4608,123 @@ async def create_certificate(
             return CLISource.single(lambda: stream.just(self.rendered_help(ctx)))
 
 
+class InfrastructureAppsCommand(CLICommand):
+    """
+    ```shell
+    apps search [pattern] [someengineering/resoto-apps]
+    app info <app_name> [someengineering/resoto-apps]
+    app install <app_name> [someengineering/resoto-apps]
+    app edit <app_name>
+    app uninstall <app_name>
+    app update <app_name>|all
+    apps list
+    app run <app_name> [--dry-run] [--validate] --config [config_name]
+    ```
+
+    - `apps search [pattern]`: Lists all apps available in https://github.com/someengineering/resoto-apps/
+    (or other repo if specified). Supports filtering by pattern.
+    - `app info <app_name>`: Show information about an app.
+    - `app install <app_name>`: Install an app.
+    - `app edit <app_name>`: Edit an app.
+    - `app uninstall <app_name>`: Uninstall an app.
+    - `app update <app_name>|all`: Update an app or all apps.
+    - `apps list`: List all installed apps.
+    - `app run <app_name> [--dry-run] [--validate] [--config <config_name>]`: Run an app.
+
+
+    ## Parameters
+    - <app_name>: The identifier of the infrastructure app.
+    - [pattern]: Pattern for searching for apps. Supports glob wildcards such as * and ?.
+    - [--dry-run]: Run the app but do not make any changes.
+    - [--validate]: Validate the app but do not run it.
+    - [--config <config_name>]: The configuration to use to run the app. Defaults to the default configuration.
+    """
+
+    @property
+    def name(self) -> str:
+        return "apps"
+
+    def info(self) -> str:
+        return "Manage infrastructure apps."
+
+    def args_info(self) -> ArgsInfo:
+        return {
+            "search": [ArgInfo(None, True, help_text="Pattern for searching for apps.")],
```

Comment:
The help text is visible via tab completion on the command.
A standard terminal has 80 characters. (the help text here is already above 40 caracters + plus name + plus position of the prompt) - I would opt to shorten the help text - e..g. `<pattern>` 
Also applies to the rest of args.